libcloud/compute/drivers/vcloud.py

```diff
@@ -1069,11 +1069,11 @@ class VCloud_5_5_Connection(VCloud_1_5_Connection):
     def _get_auth_headers(self):
         """Compatibility for using v5.5 of the API"""
         auth_headers = super(VCloud_5_5_Connection, self)._get_auth_headers()
-        auth_headers["Accept"] = "application/*+xml;version=5.5"
+        auth_headers["Accept"] = 'application/*;version=32.0'
```

Comment:
Was that change intentional? Since it looks like test case is now failing with this change - https://github.com/apache/libcloud/runs/5153389792?check_suite_focus=true#step:11:20321.

If it's not intentional, I will revert it so the tests pass and I can merge the PR.

Reply:
For now, to get the tests pass I will revert this change + change to the tests, but if this change is needed, please open a separate PR for it.

Reply:
it was intentional

Reply:
If the change was intention, please open a separate PR for that change and please also include more information / background - why is that change needed, some references, etc.